### DISCUSSION TITLE
Broken link to Policies

### DISCUSSION THREAD

**Discussion by dmarcoux** [dotnet/aspnet-api-versioning#1092] [Q&A] [answered]
Hello,

while reading some pages from the wiki, I found this broken link referring to _Policies_: https://github.com/dotnet/aspnet-api-versioning/wiki/Versioning-Policies. It's at least twice on this page: https://github.com/dotnet/aspnet-api-versioning/wiki/API-Versioning-Options

I believe it should be changed to https://github.com/dotnet/aspnet-api-versioning/wiki/Version-Policies. I would gladly do it, but I don't have sufficient permissions for this.

Thank you for this project, it's awesome!

**Comment by commonsensesoftware** [comment, marked as answer]
Thanks for point that out. Those links should be fixed now. Let me know if you find others
